### DISCUSSION TITLE
Do you plan to expose the parser? Or is there a way to use it from JavaScript?

### DISCUSSION THREAD

**Discussion by carloslfu** [biomejs/biome#427] [Q&A] [answered]
Biome's parser is awesome! And, I'd like to know if you plan to make it usable from the JS side. I explored the JS APIs but couldn't find a way to do it.

My use case is as a replacement for `acorn`/`acorn-jsx` which doesn't support error recovery. `acorn` alone does but `acorn-jsx` doesn't due to the complexity of it.

**Comment by ematipico** [comment, marked as answer]
We are working on publishing the crates https://github.com/biomejs/biome/issues/286

**Reply by carloslfu** [reply]
Nice! Thanks!
